package.json

```diff
@@ -0,0 +1,57 @@
+{
+  "name": "bichard7-next-ui",
+  "version": "0.1.0",
+  "private": true,
+  "scripts": {
+    "dev": "next dev -p 4080",
+    "build": "next build",
+    "start": "next start -p 4080",
+    "lint": "next lint",
+    "lint:fix": "eslint . --fix",
+    "cypress:open": "cypress open",
+    "cypress:run": "cypress run",
+    "cypress:run:file": "cypress run --spec",
+    "test:unit": "jest --testPathIgnorePatterns=\"integration\" \"e2e\" --runInBand",
+    "test:unit:update": "npm run test:unit -- -u",
+    "test:integration": "jest --testPathPattern=\"integration\" --runInBand",
+    "test:ui": "./scripts/run-test-ui.sh",
+    "install:hooks": "./scripts/install-git-hooks.sh",
+    "update-deps": "ncu -u && npm install"
```

Comment:
`ncu` is a tool provided by the [`npm-check-updates`](https://github.com/ministryofjustice/bichard7-next-user-service/blob/main/package.json#L82) package - which isn't listed as a dependency for this package. So I expect if you try running the `update-deps` command you've defined here (`npm run update-deps`), it'll fail 😅 

A cheeky `npm install --save-dev npm-check-updates` should do the trick!